views-thymeleaf/src/main/java/io/micronaut/views/thymeleaf/LinkBuilder.java

```diff
@@ -32,6 +34,13 @@
 @Singleton
 public class LinkBuilder extends StandardLinkBuilder {
 
+    @Inject
+    @Nullable
+    @Property(name = "micronaut.server.context-path")
+    private String contextPath;
```

Comment:
Could you use constructor injection, private field injection is a bit of an anti-pattern.

Reply:
In that case there would still be code somewhere using this class without context path, or did I miss something 😅